tests/models/grounding_dino/test_modeling_grounding_dino.py

```diff
@@ -115,7 +152,8 @@ def prepare_config_and_inputs(self):
         pixel_values = floats_tensor([self.batch_size, self.num_channels, self.image_size, self.image_size])
         pixel_mask = torch.ones([self.batch_size, self.image_size, self.image_size], device=torch_device)
 
-        input_ids = ids_tensor([self.batch_size, self.max_text_len], self.num_labels)
+        input_ids = torch.tensor([101, 3869, 1012, 11420, 1012, 1012, 102])
+        input_ids = input_ids.unsqueeze(0).expand(self.batch_size, -1)
```

Comment:
Otherwise tests that use `labels` and, therefore, compute a loss would complain as `build_label_maps` ([here](https://github.com/EduardoPach/transformers/blob/fix-grounding-dino-loss/src/transformers/models/grounding_dino/modeling_grounding_dino.py#L2916)) would return `None`

Reply:
Perhaps you can add a comment to explain this